src/Main.js

```diff
@@ -36,6 +37,16 @@ class Main extends React.Component {
     this.setState({clickedPosterID: id})
   }
 
+  searchMovies = (event) => {
+    const { value } = event.target
+    const foundMovies = this.state.movieData.filter(movie => {
+      if(movie.title.toLowerCase().includes(value.toLowerCase())) {
```

Comment:
Good call to have .toLowerCase() to make sure capitalization doesn't limit searches!